assignments/prototypes.js

```diff
@@ -15,13 +15,33 @@
   * destroy() // prototype method -> returns the string: 'Object was removed from the game.'
 */
 
+function GameObject(object) {
+  this.createdAt = object.createdAt
+  this.dimensions = object.dimensions
+}
+
+GameObject.prototype.destroy = function () {
+  return "Object was removed from the game.";
```

Comment:
The instructions aren't completely clear hear, but if you look down at the console log that tests the destroy() method:
```
console.log(swordsman.destroy()); // Sir Mustachio was removed from the game.
```
You can see that this string is supposed to contain the object's name. You can fix this by using a template literal and `${this.name}`.